tmail-backend/tmail-third-party/openpaas/pom.xml

```diff
@@ -103,8 +107,25 @@
             <artifactId>ez-vcard</artifactId>
             <version>0.12.1</version>
         </dependency>
+        <dependency>
+            <groupId>org.wiremock</groupId>
+            <artifactId>wiremock</artifactId>
+            <version>3.10.0</version>
+        </dependency>
     </dependencies>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.eclipse.jetty</groupId>
+                <artifactId>jetty-bom</artifactId>
```

Comment:
Please put a comment on why it is needed?

Why does wiremock do not pull this itself?

Reply:
It turns out that declaring the spark-core dependency explicitly is not necessary at this location as it is being imported transitively by `org.apache.james:james-server-mailets-integration-testing`. After removing it, the dependency conflicts resolution algorithm from maven selects version 11 of jetty-server which is good for our Wiremock and bad for spark-core; apparently because the two versions are binary compatible (I don't know for sure but it explains why the code compiled fine).

It's worth noting that the conflict between spark-core and wiremock is confined to the test scope.